tests/unit/Model/ResponseTest.php

```diff
@@ -56,10 +59,12 @@ public function provideResponseData(): array
         $skippedCommandResponse = (object) ['status' => CommandResponse::STATUS_SKIPPED, 'message' => '', 'data' => []];
 
         return [
-            'empty response data' => [0, 0, 0, 0, []],
-            'multiple successful commands' => [2, 2, 0, 0, [$okCommandResponse, $okCommandResponse]],
-            'multiple failed commands' => [2, 0, 2, 0, [$failedCommandResponse, $failedCommandResponse]],
-            'multiple skipped commands' => [2, 0, 0, 2, [$skippedCommandResponse, $skippedCommandResponse]],
+            'empty response data' => [0, 0, 0, 0, [], true],
```

Comment:
Maybe `skipped and failed` case should be covered? 🤔

Reply:
In term of combinatorics - yes, but will it actually test anything - any path - that is not already tested?

There is just one combination missing to achieve all possible input combinations in this case, so here is it non-issue. However in case of 4 inputs, would you test all 16 combinations? (By manually written datasets in provider) 

🤔